### PR TITLE
Use default env vars for selected networks

### PR DESCRIPTION
**Pull request by lukaszcl**
Use EVM_URLS, EVM_KEYS, EVM_HTTP_URLS env vars if network specific env vars not provided

### CODE REVIEW

File: networks/known_networks.go [hunk 1]
```diff
@@ -637,25 +638,52 @@ func MustGetSelectedNetworksFromEnv() []blockchain.EVMNetwork {
 			wsEnvVar := fmt.Sprintf("%s_URLS", networkKeys[i])
 			wsEnvVal := os.Getenv(wsEnvVar)
 			if wsEnvVal == "" {
-				panic(errors.Errorf("set %s env var", wsEnvVar))
+				// Get default value
```
Comment (kalverra) [inline networks/known_networks.go:641]: Worth a warning log to indicate this is what you're doing, something like `log.Warn().Msgf("%s not set, defaulting to EVM_URLS", httpEnvVar)`

File: networks/known_networks.go [hunk 2]
```diff
@@ -637,25 +638,52 @@ func MustGetSelectedNetworksFromEnv() []blockchain.EVMNetwork {
...
+				wsUrls = strings.Split(wsEnvVal, ",")
 			}
-			wsUrls = strings.Split(wsEnvVal, ",")
 
 			// Get network RPC HTTP URL from env var
 			httpEnvVar := fmt.Sprintf("%s_HTTP_URLS", networkKeys[i])
 			httpEnvVal := os.Getenv(httpEnvVar)
 			if httpEnvVal == "" {
-				panic(errors.Errorf("set %s env var", httpEnvVar))
+				// Get default value
```
Comment (kalverra) [inline networks/known_networks.go:658]: See above

File: networks/known_networks.go [hunk 3]
```diff
@@ -637,25 +638,52 @@ func MustGetSelectedNetworksFromEnv() []blockchain.EVMNetwork {
...
+				httpUrls = strings.Split(httpEnvVal, ",")
 			}
-			httpUrls = strings.Split(httpEnvVal, ",")
 
 			// Get network wallet key from env var
 			walletKeysEnvVar := fmt.Sprintf("%s_KEYS", networkKeys[i])
 			walletKeysEnvVal := os.Getenv(walletKeysEnvVar)
 			if walletKeysEnvVal == "" {
-				panic(errors.Errorf("set %s env var", walletKeysEnvVar))
+				// Get default value
```
Comment (kalverra) [inline networks/known_networks.go:675]: See above

**Review by kalverra — CHANGES_REQUESTED** [review]

File: networks/known_networks.go (hunk 3 above)
Comment (lukaszcl) [reply networks/known_networks.go:677]: @kalverra added in 61ddd40688da4b2e171753d84dd5ca2c29ef03e2

File: networks/known_networks.go (hunk 1 above)
Comment (lukaszcl) [reply networks/known_networks.go:641]: @kalverra good idea. Added in 61ddd40688da4b2e171753d84dd5ca2c29ef03e2

**Review by kalverra — APPROVED** [review]

**Pull request merged**